config/laravel-swagger.php

```diff
@@ -85,12 +84,9 @@
         [
             'appVersion' => '1.0.0',
 
-            'basePath' => '/v1',
+            'host' => env('APP_URL'),
 
-            'schemes' => [
-                // 'http',
-                // 'https',
-            ],
+            'basePath' => '/',
```

Comment:
didn't want to assume the basePath of the application, especially since I don't know if most people using this will have their application be versioned but it's there as an option for them